packages/p2p/src/components/my-ads/my-ads-table.jsx

```diff
@@ -61,6 +62,25 @@ const MyAdsTable = () => {
                         />
                     </div>
                 )}
+                {floating_rate_store.change_ad_alert && (
+                    <div className='p2p-my-ads__warning'>
+                        <HintBox
+                            icon='IcAlertWarning'
+                            message={
+                                <Text as='p' size='xxxs' color='prominent' line_height='xs'>
+                                    <Localize
+                                        i18n_default_text={`Floating rates are enabled for ${
+                                            general_store.client.local_currency_config.currency || '-'
+                                        }. Ads with fixed rates will be deactivated. Switch to floating rates by ${
+                                            floating_rate_store.fixed_rate_adverts_end_date || '-'
+                                        }`}
+                                    />
```

Comment:
Localize doesn't accept `` string operator, please use values.

Reply:
The text does render

![Screen Shot 2022-03-16 at 3 32 35 PM](https://user-images.githubusercontent.com/98398322/158580961-26cc44cc-8e42-42ca-aeb2-62126b2216d4.png)

Reply:
I will make the change